include/openssl/core_names.h

```diff
@@ -40,6 +40,22 @@ extern "C" {
  */
 #define OSSL_PROV_PARAM_MODULE_FILENAME "module-filename"
 
+/*
+ * A string for a self test callback's phase.
+ * Type: OSSL_PARAM_UTF8_STRING
+ */
+#define OSSL_PROV_PARAM_SELF_TEST_PHASE  "st-phase"
+/*
+ * A string for a self test callback's type.
+ * Type: OSSL_PARAM_UTF8_STRING
+ */
+#define OSSL_PROV_PARAM_SELF_TEST_TYPE   "st-type"
+/*
+ * A string for a self test callback's description.
+ * Type: OSSL_PARAM_UTF8_STRING
+ */
+#define OSSL_PROV_PARAM_SELF_TEST_DESC   "st-desc"
+
```

Comment:
I like these comments, maybe we should do this consistently.

Reply:
I just simplified them to match the others.